velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -712,6 +720,7 @@ TEST_F(StringImplTest, pad) {
   runTest(
       "abcd\xff \xff ef", 11, "0", "0abcd\xff \xff ef", "abcd\xff \xff ef0");
   runTest("abcd\xff ef", 6, "0", "abcd\xff ", "abcd\xff ");
+  runTestUserError(/*string=*/"\u4FE1", /*size=*/6, /*padString=*/"\xBF\xBF");
```

Comment:
Could you add a comment here explaining this is to test the case where the padString is simply a bunch of unicode continuation bytes which effectively has length 0.

Reply:
Done.